src/scvi_v2/_model.py

```diff
@@ -939,20 +1062,20 @@ def per_sample_inference_fn(pair):
                     return inference_fn(rngs, cf_sample)
 
                 eps_ = jax.lax.transpose(jax.lax.map(per_sample_inference_fn, (stacked_rngs, cf_sample)), (1, 0, 2))
-            eps_ = eps_[:, donor_mask]
-            eps = (eps_ - eps_.mean(axis=1, keepdims=True)) / (1e-6 + eps_.std(axis=1, keepdims=True))  # over samples
+            eps_ = eps_[:, :, donor_mask]
```

Comment:
bug here, the dimension of donors is the third (and not second) dimension
```suggestion
            eps_ = eps_[:, :, donor_mask]
```

Reply:
I don't get it. I can't see a change in the code?

Reply:
nevermind, everything seems ok